src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -2,6 +2,7 @@ import {Location} from 'history';
 
 import {t} from 'app/locale';
 import {NewQuery} from 'app/types';
+import EventView from 'app/views/eventsV2/eventView';
```

Comment:
Maybe we should move this outside of `views`? Could `app/utils/discover/eventView` work?